src/applications/mhv/medical-records/containers/ConditionDetails.jsx

```diff
@@ -37,6 +37,7 @@ import { txtLine } from '../../shared/util/constants';
 const ConditionDetails = props => {
   const { runningUnitTest } = props;
   const record = useSelector(state => state.mr.conditions.conditionDetails);
+  const condition = useSelector(state => state.mr.vaccines.conditionList);
```

Comment:
State should come from `state.mr.conditions.conditionsList`. I would also call the variable `conditionList` or `conditions` to avoid confusion.

Reply:
This issue is fixed.